### ISSUE TITLE
#boycottrealmeme

### ISSUE THREAD

**Issue by randomuser10169** [realme-kernel-opensource/realme3-3i-AndroidQ-kernel-source#7]
#BoycottRealmeme
u gey af guys dont even bother to provide proper source for devices and realme 1 and u1 didnt even get pie kernel source,
bsdwalo kuch sharam kro, khud bolte ho hum costomer centric, lawda lassan ho...lekin tum lavde ke customer centric ho, ainda se india me esi tatti devices mat bechna bc jiski kernel source lawde jaise engineers bnate he

**Comment by eventhorizon-21** [comment]
**Just realme things**

_"here! take this 2000₹ with a cup of cocaine and write kernel source for us"_

**Comment by TipzTeam** [comment]
Agreed, although I don't own this device, they should improve their kernel quality
